webapp/src/main/webapp/services/newsTargetingService.js

```diff
@@ -14,7 +14,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-import {newsConstants} from '../../js/newsConstants.js';
+import {newsConstants} from '../js/newsConstants.js';
```

Comment:
keep NewsListService.js only for news list services and move those or taregting in newsTargetingService.js